server/historian/packages/historian-base/src/services/restGitService.ts

```diff
@@ -83,6 +84,9 @@ export class RestGitService {
 			);
 			defaultHeaders.Authorization = `Basic ${token.toString("base64")}`;
 		}
+		if (this.simplifiedCustomData) {
+			defaultHeaders[Constants.SimplifiedCustomData] = this.simplifiedCustomData;
```

Comment:
Since you put it in the header, just be aware of the size of this new data. You can add implementation in FRS about what field to keep in this field. Another question is that why not put it in request body instead of headers?

Reply:
Discussed offline. The size of this data will be small.